### DISCUSSION TITLE
Dragging files/folder outside of SpaceDrive window, what's up?

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2449] [General]
I noticed, trying to drag files or folder from a SpaceDrive Window to an open File Explorer window, the icons of the selection set vanish after cursor hits the border of the SpaceDrive window. Same when trying to drag files and folders to the desktop.

Is this a current limitation due to being in alpha (that's fine an expected a this stage) or intended design?

**Comment by utkubakir** [comment]
Limitation, drag and drop outside is not implemented yet.
